src/test/java/lotto/unit/controller/LottoGameControllerTest.java

```diff
@@ -0,0 +1,62 @@
+package lotto.unit.controller;
+
+import static camp.nextstep.edu.missionutils.test.Assertions.assertSimpleTest;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import camp.nextstep.edu.missionutils.test.NsTest;
+import lotto.controller.LottoGameController;
+import lotto.view.OutputMessage;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class LottoGameControllerTest extends NsTest {
+    private static final String LOTTO_COST = "1000";
+    private static final String WINNING_NUMBERS = "1,2,3,4,5,6";
+    private static final int REPRINTED_COUNT_NUMBER = 2;
+
+    private static LottoGameController createLottoGameController() {
+        return new LottoGameController();
+    }
+
+    private int countMatchMessage(String output, String message) {
+        return (output.length() - output.replace(message, "").length()) / message.length();
+    }
+
+    @ParameterizedTest
```

Comment:
`@ParameterizedTest`를 사용하면서도 이름을 각각 부여할 수 있다는 사실 알고 계셨나요?!
```suggestion
    @ParameterizedTest(name = "\"{0}\"을 입력하면 입력 안내 메시지를 다시 출력한다")
```

Reply:
오오..꿀팁 감사합니다!😊